### ISSUE TITLE
Support for ROMs on USB Flash drive

### ISSUE THREAD

**Issue by LogicalJoe** [calc84maniac/tiboyce#130]
Going to be frank, ROMs are big.  Wouldn't it be nice if (with a little bit of overhead) you could save all that space and have the ROM on an external storage device?  I'm not sure how possible this is with arbitrary calls and code already being optimized for having most of the data on-hand but I decided might as well make an issue on this.

Not sure if this should be more of a Cesium USB API issue, but I kinda like this being a stand-alone program.

**Comment by calc84maniac** [comment]
You're right that all the data has to be directly on hand. This is also why I don't support compressed ROMs. I've considered it before, but games tend to frequently access data from all over the ROM, to the point that caching uncompressed data (or data from a flash drive) likely wouldn't work all that well with the limited amount of RAM available.

It's not out of the question for me to at least attempt something like this in the future, even with the possibly poor performance, just to have the possibility of running 4MB Game Boy Color ROMs. But for now, the best option is to use something like Cesium to copy the ROM files to and from your calculator.
